Extension/src/LanguageServer/client.ts

```diff
@@ -335,6 +335,20 @@ interface DidChangeConfigurationParams extends WorkspaceFolderParams {
     settings: any;
 }
 
+interface GetFoldingRangesParams {
```

Comment:
Should this inherit from AbortRequestsParams?

Reply:
It could, to consolidate the common field.  But, they aren't technically related.  The are params for different messages.